default/stringtables/en.txt

```diff
@@ -899,6 +899,12 @@ Error when reading save file.
 UNABLE_TO_SAVE_NOW_TRY_AGAIN
 Unable to save during AI processing. Try again when AIs have finished turns.
 
+SAVE_GAME_IN_PROGRESS
+A save game is in progress.  Click Stop Waiting to abandon the save.
+
+STOP_WAITING
+Stop waiting.
```

Comment:
I've split the messages in separate ones for return to main menu and exiting. 
`Return to Main Menu without saving.`
and
`Exit FreeOrion without saving.`

This is the same phrasing as the `RETURN_TO_INTRO` message.